src/ert/storage/local_ensemble.py

```diff
@@ -485,11 +534,50 @@ def save_parameters(
 
         dataset.expand_dims(realizations=[realization]).to_netcdf(path, engine="scipy")
 
+        if os.path.exists(self._path / f"{group}.nc"):
+            os.remove(self._path / f"{group}.nc")
+
     @require_write
     def save_response(self, group: str, data: xr.Dataset, realization: int) -> None:
         if "realization" not in data.dims:
             data = data.expand_dims({"realization": [realization]})
+
         output_path = self._realization_dir(realization)
         Path.mkdir(output_path, parents=True, exist_ok=True)
 
         data.to_netcdf(output_path / f"{group}.nc", engine="scipy")
+
+        if os.path.exists(self._path / f"{group}.nc"):
```

Comment:
It will happen if someone saves a parameter after they are accessed. So it supposes that all responses/parameters are written to storage before they are accessed. If a re-write happens, it deletes the combined one which will trigger a recomputation. Will add a log statement to catch this, which I think should(ideally) be solved by ensuring that writes are (mostly) done before reads.